src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/architecture.mdx

```diff
@@ -14,20 +14,20 @@ metaDescription: Implementation details and data flow for New Relic Lambda monit
 
 There are several important parts to Lambda monitoring:
 
-1. Your function
-2. The New Relic agent or SDK
-3. The New Relic Lambda Extension
-4. New Relic's backend
-5. The New Relic One Lambda UI
+1. [Your function](#your-function)
+2. [The New Relic agent or SDK](#agent-sdk)
+3. [The New Relic Lambda Extension](#lambda-extension)
```

Comment:
Some users use the older method of sending telemetry to NR--the aws-log-ingestion function, which is triggered by a CloudWatch log subscription. There are downsides--it incurs slightly higher costs (an invocation of the ingestion function in addition to the invocation of the function the customer has instrumented, and the CW log subscription), and it's slightly slower. But if a customer has strict requirements that prevent them from allowing outbound VPC traffic from their Lambda function to NR (for example), then they may still need to use this older method instead of the extension.